### DISCUSSION TITLE
home-assistant-glow & UM tinypico

### DISCUSSION THREAD

**Discussion by adandl** [klaasnicolaas/home-assistant-glow#230] [Q&A]
Hello All!

Attention - noob question.
I want to run glow on a UM tinypico - which apparently does not have a pinout for GPIO13.

see -> https://www.tinypico.com/gettingstarted
Q: how can I change the GPIO13 for receiving the pulse to a different GPIO?
Thanks
Albert

**Comment by klaasnicolaas** [comment]
You can copy the [home_assistant_glow.yaml](https://github.com/klaasnicolaas/home-assistant-glow/blob/main/home_assistant_glow.yaml#L15) file to your own esphome folder, change the `pulse_pin` and compile a new firmware version.

**Reply by adandl** [reply]
Hi - 
Thank you very much for your quick reply!
Have you thought about making this configurable in the application? Is that an option?
I'd rather stay at your most recent version than forking it ;-)

**Reply by klaasnicolaas** [reply]
Not quite sure what you mean by configurable, but this is something you really have to adjust yourself.
